.github/workflows/bot-bump-sglang-version.yml

```diff
@@ -26,6 +26,10 @@ jobs:
         with:
           python-version: '3.10'
 
+      - name: Install Python dependencies
+        run: |
+          pip install tomli
```

Comment:
Don't install dependencies in yml, let's update bash script instead

Reply:
Do we want to update scripts/ci/ci_install_dependency.sh? That file is very large and most of them are not relevant. Or I can create a different shell script

Reply:
Can we update in ci_install_dependency.sh

Reply:
ci_install_dependency.sh is mostly used in GPU nodes, if I run ci_install_dependency.sh it will complain nvidia-smi not found, and I feel like we don't have to refactor that file for a simple change in these two workflows